src/graph.js

```diff
@@ -21,6 +21,10 @@ class Graph {
 		this._relationshipTypes = []; 	// List of relation types.
 		this._properties = [];        	// List of properties.
 
+		this._labelsPromise = undefined;
```

Comment:
Add comments for _labelsPromise, _propertyPromise and _relationshipPromise explaining what are they used for,
similar to any other class member.